Documentation/ReleaseNotes/3.12.md

```diff
@@ -1165,10 +1165,10 @@ Changes in this Release
         STYLE: according to kwstyle.
 
     Code/Common/itkImageBase
-        BUG: 8490: ImageBase::Initialized method modifies it's MTime requiring pipelines to re-executed. Added InitializeBufferedRegion method which does not call Modify as a protected method
+        BUG: 8490: ImageBase::Initialized method modifies its MTime requiring pipelines to re-executed. Added InitializeBufferedRegion method which does not call Modify as a protected method
```

Comment:
I am not sure whether we should be updating old release notes.

Reply:
@dzenanz  I think we should update release notes for grammatical changes.